index.html

```diff
@@ -1,44 +1,225 @@
+                    </li>
+                    <li><img src="images/header/javas.png"JavaScript"></li>
+                    <li><img src="images/header/ht.png" alt="Html"></li>
+                    <li><img src="images/header/cs.png" alt="Css"></li>
+                    <li class="my_skill_two">
+                        <p>Frameworks</p>
+                        <a href="#"><img class="social" src="images/header/desktop/Union.png" alt="L_cursor"></a>
+                    </li>
+                    <li><img class="the_line" src="images/header/line.png" alt="line"></li>
+                    <li class="my_skill_one">
+                        <p>Skills</p>
+                        <a href="#"><img src="images/header/desktop/Union.png" alt="L_cursor"></a>
+                    </li>
+                    <li><img class="divider_line" src="images/header/line.png" alt="line"></li>
+                </ul>
+            </div>
+            <div class="my_skills_desktop">
+                <ul class="my_skills_list_desktop">
+                    <li class="skill_language">
+                        <p>Languages</p>
+                        <a href="#"><img class="social" src="images/header/arrow.png" alt="Cursor-Down"></a>
+                    </li> <br>
+                    <li class="language_expanded">
+                        <img src="images/header/javas.png" alt="javascript">
+                        <img src="images/header/ht.png" alt="html">
+                        <img src="images/header/cs.png" alt="css">
+                    </li> <br>
+                    <li class="skill_unexpanded">
+                        <p>Framework</p>
+                        <a href="#"> <img src="images/header/desktop/Union.png" alt="left-cursor"> </a>
+                    </li>
+                    <li>
+                        <img class="divider_line" src="images/header/line.png" alt="line">
+                    </li> <br>
+                    <li class="skill_unexpanded">
+                        <p>Skills</p> <a href="#"><img src="images/header/desktop/Union.png" alt="left-cursor"></a>
+                    </li>
+                    <li><img class="divider_line" src="images/header/line.png" alt="line"></li>
+                </ul>
+            </div>
+        </section>
+    </main>
+    <footer id="section_footer" class="foo_ter">
+        <div class="footer_background">
+            
+            <form class="the_form" action="https://formspree.io/f/xvoyankb" method="post">
+                <input type="text" id="name" name="name" placeholder="user name" maxlength="30" required>
+                <input type="email" id="email" name="email" placeholder="example@email.com" required>
+                <textarea type="text" id="Message" name="message" placeholder="write your message here" maxlength="500" required></textarea>
+                <button class="for_button" type="submit">Get it Touch</button>
+            </form>
         </div>
-    </div>
+    </footer>
```

Comment:
- [ ] Kindly replace the `footer` tag with `section` tag since this section is for contact and it's not a footer. 
- [ ] Kindly try to follow the design on Figma as much as possible. The `contact` section has a `background-image`, it also has a header and a paragraph before the form. Lastly, it has a `border-radius` on the top left corner 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194709427-7a04af3e-49a9-4e78-8f90-30dd415d6fbc.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194709440-3b04212c-7d8c-48be-a58a-48c796111d2c.png)